pkg/distro/rhel9/distro_test.go

```diff
@@ -498,6 +501,43 @@ func TestDistro_ManifestError(t *testing.T) {
 	}
 }
 
+// Check that Manifest() function returns an warning when FIPS
+// customization is enabled and the host is not FIPS
+func TestDistro_ManifestFIPSWarning(t *testing.T) {
+	r8distro := rhel9.New()
```

Comment:
copy pasta typo :)

Unrelated, but my brain sees these tests with almost exactly the same code and wishes it could be a function. But not really asking for that.

Reply:
@bcl  I moved the tests to `pkg/distro/distro_test.go` and simplified some things. I hope this is ok with you.